modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -141,27 +145,26 @@ def offset(
         quotechar: bytes = b'"',
         is_quoting: bool = True,
     ):
-        """Moves the file offset at the specified amount of bytes.
+        """
+        Move the file offset at the specified amount of bytes.
 
         Parameters
         ----------
-        f: file-like object
+        f : file-like object
             file handle that should be used for offset movement.
-        offset_size: int
+        offset_size : int
             Number of bytes to read and ignore.
-        quotechar: bytes, default b'"'
+        quotechar : bytes, optional. Default b'"'
```

Comment:
@prutskov @YarShev @anmyachev what is the optional format we agreed upon?.. I think it was the one Alexander has changed now :)

Reply:
We agreed on this [use](https://github.com/modin-project/modin/issues/2934#issuecomment-815904375). @anmyachev , does the script check it?

Reply:
Yes, it checks. The only valid case right now is `[type], default: [value]`

Reply:
Yes, there are 3 equivalent options written there, but I think that if we make this place stricter, then it will not be worse